hbase-client/src/main/java/org/apache/hadoop/hbase/client/SecureBulkLoadClient.java

```diff
@@ -56,7 +57,7 @@ public String prepareBulkLoad(final Connection conn) throws IOException {
     try {
       ClientServiceCallable<String> callable =
         new ClientServiceCallable<String>(conn, table.getName(), HConstants.EMPTY_START_ROW,
-          this.rpcControllerFactory.newController(), PRIORITY_UNSET) {
+          this.rpcControllerFactory.newController(), PRIORITY_UNSET, Collections.emptyMap()) {
```

Comment:
this class is constructed with a Table, so might as well pass in Table.getRequestAttributes()